js/okx.js

```diff
@@ -2442,7 +2442,7 @@ module.exports = class okx extends Exchange {
         const marketId = this.safeString (order, 'instId');
         const symbol = this.safeSymbol (marketId, market, '-');
         const filled = this.safeString (order, 'accFillSz');
-        const price = this.safeString2 (order, 'px', 'ordPx');
+        const price = this.safeStringN (order, [ 'px', 'ordPx', 'slOrdPx', 'tpOrdPx' ]);
```

Comment:
@sc0Vu can we have `px` and `slOrderPx`/`tpOrderPx` simultaneously?

Reply:
Do you mean they'll exist in the same order? Not sure about this, didn't see in their documentation.

Reply:
@sc0Vu yes, we have to make sure that we will not "hide" any value

Reply:
in my understanding, `tpOrdPx` is only set for OCO orders - and is the take profit price (in such a case, the order would have 4 prices).

Stop order (conditional)
![image](https://user-images.githubusercontent.com/5024695/201177305-52b1ef13-b5c9-4ae0-9edf-1cc4f17eea76.png)

(OCO order - type "oco")
![image](https://user-images.githubusercontent.com/5024695/201177347-db149a9f-98c0-41d8-a003-2936c9b9b258.png)

Reply:
You're right. We should set these value for stopPrice.